tests/e2e/cucumber/environment/index.ts

```diff
@@ -39,6 +42,7 @@ BeforeAll(async function () {
 
 Before(function (this: World, { pickle }: ITestCaseHookParameter) {
   this.feature = pickle
+  featureName = camelCase(this.feature.name)
```

Comment:
```suggestion
const  featureName = camelCase(this.feature.name)
```
Why not use a const here?

Reply:
I created above `export let featureName = ''` 
and then wrote the values into it. 
is that wrong?

Reply:
Not necessary since you don't change the value later on, so removing the `export let featureName = ''` and changing line 45 to `const featureName = camelCase(this.feature.name)` should be fine
`let` is necessary because you change it to keep track of the retries ;)

Reply:
I'm exporting featureName
And get featureName in actor.ts. That's what I asked you to help me with. Move correct featureName from index.ts to actor.ts

when i will remove `export let featureName = ''` then i don't get featureName in actor.ts

Reply:
![image](https://user-images.githubusercontent.com/84779829/153196260-6e97f03f-d973-4fda-85da-c100ebd4207a.png)